### ISSUE TITLE
Android-Iconics

### ISSUE THREAD

**Issue by dexion** [mikepenz/AboutLibraries#182]
Android-Iconics doesn't work in AboutActivity :(
```java
new LibsBuilder()
                .withActivityStyle(Libs.ActivityStyle.LIGHT_DARK_TOOLBAR)
                .withAboutIconShown(true)
                .withAboutVersionShown(true)
                .withAboutDescription("{faw-android} This is a small sample.<br /><b>You can style this with html markup :D</b>")
                .start(context);
```

`{faw-android}` shown like text, not icon. In other `TextView`s this alias work perfectly

**Comment by mikepenz** [comment]
@dexion good idea. I'll probably add the `Iconics-core` as dependency to the AboutLibraries, so this is also possible.

**Comment by Rainer-Lang** [comment]
Good idea

**Comment by dexion** [comment]
Thanks for that! But now html markup is broken: at least `<b>` tag doesn't work (shown like regular text, not bold)

**Comment by mikepenz** [comment]
@dexion ok that's a problem with `Android-Iconics` if i apply the icon fonts, i will create new spans on the text. As the `Html.fromText()` already created the spans, i will overwrite them. That will be a more complex task. will look into it.

**Comment by mikepenz** [comment]
@dexion ok this issue is now also fixed in v5.3.6

**Comment by dexion** [comment]
Great! Thanks!
